symantec-messaging-gateway/operations.py

```diff
@@ -297,13 +306,11 @@ def audit_logs_search(self, config, params):
             endpoint = AUDIT_LOGS + '$search.flo'
             resp = self._make_request(endpoint, 'post', data=search_params)
             json_response = html_to_json(resp.text)
-            logger.info(json.dumps(json_response, indent = 3))
+            logger.debug(json.dumps(json_response, indent=3))
```

Comment:
Debug? Do we really need it?

Reply:
In case some some logs are missing we need to take the dump of the output.  so generally we kept the debug statements.